### ISSUE TITLE
liquidsoap exits on USR1

### ISSUE THREAD

**Issue by liquidpeter** [savonet/liquidsoap#363]
I try to send USR1 signal to liquidsoap to reopen any logs.
In case there are no logs configured liquidsoap exits with exit code 138.
As USR1 is used in liquidsoap to reopen logs, I would expect liquidsoap to continue working and maybe output an warning, but not to exit.

example: 
```
$ liquidsoap 'out(sine())' &
$ ps -afe | grep liquidsoap
peter     4816  4705  1 19:01 pts/0    00:00:00 liquidsoap out(sine())
peter     4838  4824  0 19:01 pts/2    00:00:00 grep liquidsoap
$ kill -USR1 4816

2016/11/17 19:01:10 [clock.wallclock_pulse:3] Streaming loop starts, synchronized by active sources.
2016/11/17 19:01:10 [mksafe:3] Switch to sine_9273.
User Defined Signal 1

$ echo $?
138
```
Best regards, Peter

**Comment by toots** [comment]
Interesting one!

**Comment by toots** [comment]
Should be fixed with savonet/ocaml-tools@df7c76a Are you able to test it?
